### PR TITLE
Added Documentation for Supabase Edge Functions

### PR DESCRIPTION
**Pull request by probir-sarkar**
image

Working properly. visit demo website of added supabase doc: https://hono-website.pages.dev/getting-started/supabase-functions

### CODE REVIEW

File: .vitepress/config.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ const sidebars = (): DefaultTheme.SidebarItem[] => [
       { text: 'AWS Lambda', link: '/getting-started/aws-lambda' },
       { text: 'Lambda@Edge', link: '/getting-started/lambda-edge' },
       { text: 'Node.js', link: '/getting-started/nodejs' },
+      { text: 'Supabase Functions', link: '/getting-started/supabase-functions' },
```
Comment (yusukebe) [inline .vitepress/config.ts:34]: No big reason, but I want Node.js to be last, so can you write it after `Lambda@Edge`?

**Comment by yusukebe** [comment]
Hi @probir-sarkar 

Thanks! I've left one comment. Check it, please!

**Comment by probir-sarkar** [comment]
@yusukebe I have changed the position of supabase doc.

**Comment by yusukebe** [comment]
@probir-sarkar 

Thanks! I'll merge this now.

**Pull request merged**